src/pages/profile/index.js

```diff
@@ -103,7 +102,7 @@ const ViewProfile = () => {
                 </div>
               </div>
               <br></br>
-              <small className="padding-field-name">First Name*</small>
```

Comment:
Should this be removed entirely?

Reply:
Do you mean the br? We've just removed it